addons/godot-firebase/auth/auth.gd

```diff
@@ -435,4 +479,10 @@ func get_clean_keys(auth_result : Dictionary) -> Dictionary:
         cleaned[key.replace("_", "").to_lower()] = auth_result[key]
     return cleaned
 
-
+func _url_decode(url : String) -> String:
+    var decoded_url : String = url.replacen("%21", "!").replacen("%23", "#").replacen("%24", "$") \
+    .replacen("%26", "&").replacen("%27", "'").replacen("%28", "(").replacen("%29", ")") \
+    .replacen("%2A", "*").replacen("%2B", "+").replacen("%2C", ",").replacen("%2F", "/") \
+    .replacen("%3A", ":").replacen("%3B", ";").replacen("%3D", "=").replacen("%3F", "?") \
+    .replacen("%40", "@").replacen("%5B", "[").replacen("%5D", "]")
+    return decoded_url
```

Comment:
Doesn't https://docs.godotengine.org/en/stable/classes/class_string.html#class-string-method-percent-decode handle this, or does it do something you don't want? Or not do a complete job? I guess this is all weird characters, hm.

Reply:
Honestly I wasn't aware of this method, I'll replace it!